android/java/org/chromium/chrome/browser/brave_leo/BraveLeoSettingsLauncherHelper.java

```diff
@@ -58,6 +59,21 @@ private static void handleVoiceRecognition(WebContents webContents, String conve
                 .startVoiceRecognition();
     }
 
+    @CalledByNative
+    private static void handleShowSoftKeyboard(WebContents webContents) {
+        WindowAndroid windowAndroid = webContents.getTopLevelNativeWindow();
+        if (windowAndroid == null) {
+            return;
+        }
+        Activity activity = windowAndroid.getActivity().get();
+        if (activity == null) {
+            return;
+        }
+        InputMethodManager imm =
+                (InputMethodManager) activity.getSystemService(Context.INPUT_METHOD_SERVICE);
+        imm.showSoftInput(activity.getWindow().getCurrentFocus(), 0);
```

Comment:
what will happen if the user has a physical keyboard attached?

Reply:
that's a good point, perhaps it will popped up, I have a physical keyboard to try it on, I'll check that.

Reply:
I verified it with a physical keyboard and it's not an issue, the soft keyboard is blocked on OS level if the physical one is connected.